### ISSUE TITLE
SQS Dead letter

### ISSUE THREAD

**Issue by goochi1** [terraform-aws-modules/terraform-aws-sqs#9]
Hi, Is there a way to creare a dead letter que in Terraform?

**Comment by bryantbiggs** [comment]
yes - you would need to setup a 2nd queue to handle this and update your upstream queue's redrive_policy attribute 

```terraform
redrive_policy = jsonencode({
    deadLetterTargetArn = aws_sqs_queue.terraform_queue_deadletter.arn
    maxReceiveCount     = 4
  })
```

**Comment by pingdhinakaran** [comment]
resource "aws_sqs_queue" "terraform_queue_deadletter" {
  name = "terraform-example-queue-deadletter"
  delay_seconds = 90
  max_message_size = 2048
  message_retention_seconds = 86400
  receive_wait_time_seconds = 10
}
